recipes/conda_gitenv/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set version = "0.2.0" %}
+
+package:
+  name: conda-gitenv
```

Comment:
So the directory name and recipe name don't match. Hence this has failed to convert. Should it be a `-` or a `_`? Could you please send a follow up PR to fix this, @bjlittle ?